airbyte-integrations/connectors/destination-rabbitmq/destination_rabbitmq/destination.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+from typing import Mapping, Any, Iterable
+
+from airbyte_cdk import AirbyteLogger
+from airbyte_cdk.destinations import Destination
+from airbyte_cdk.models import AirbyteConnectionStatus, ConfiguredAirbyteCatalog, AirbyteMessage, Status, Type
+
+import json
+import pika
+from pika.adapters.blocking_connection import BlockingConnection
+from pika.spec import BasicProperties
+from pika.credentials import PlainCredentials
+
+_DEFAULT_PORT = 5672
+
+
+def create_connection(config: Mapping[str, Any]) -> BlockingConnection:
+    username = config.get('username')
+    password = config.get('password')
+    virtual_host = config.get('virtual_host')
+    params = pika.ConnectionParameters(
+        host=config['host'],
+        port=config.get('port') or _DEFAULT_PORT
+    )
+    if username and password:
+        params.credentials = PlainCredentials(username=username, password=password)
+    if virtual_host:
+        params.virtual_host = virtual_host
+    return BlockingConnection(params)
```

Comment:
```suggestion
    '''
    host = config.get('host')
    port = config.get('port')
    username = config.get('username')
    password = config.get('password')
    virtual_host = config.get('virtual_host')
    ssl_enabled = config.get('ssl', True)
    amqp_protocol = 'amqp'
    host_url = host

    if ssl_enabled:
        amqp_protocol = 'amqps'

    if port:
        host_url = host + ':' + port

    params = pika.URLParameters(f'{amqp_protocol}://{username}:{password}@{host_url}/{virtual_host}')
    return BlockingConnection(params)
```

why not use the amqp url? this way I was able to connectot to RabitMQ cloud service.

Reply:
Thank you for this suggestion. I'm not familiar with the Python dependency for RabbitMQ, so I didn't know you could connect this way with it. I've updated it in b3f2cee.